demos/face_detection_mtcnn_demo/python/face_detection_mtcnn_demo.py

```diff
@@ -0,0 +1,281 @@
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                      help='Show this help message and exit.')
+    args.add_argument("-i", "--input",
+                      help="Required. Path to a test image file.",
+                      required=True, type=str)
+    args.add_argument("-m_p", "--model_pnet",
+                      help="Required. Path to an .xml file with a pnet model.",
+                      required=True, type=Path, metavar='"<path>"')
+    args.add_argument("-m_r", "--model_rnet",
+                      help="Required. Path to an .xml file with a rnet model.",
+                      required=True, type=Path, metavar='"<path>"')
+    args.add_argument("-m_o", "--model_onet",
+                      help="Required. Path to an .xml file with a onet model.",
+                      required=True, type=Path, metavar='"<path>"')
+    args.add_argument("-th", "--threshold",
+                      help="Optional. The threshold to define the face is recognized or not.",
+                      type=float, default=0.6, metavar='"<num>"')
+    args.add_argument("-d", "--device",
+                      help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL, MYRIAD or HETERO: is "
+                           "acceptable. The sample will look for a suitable plugin for device specified. Default "
+                           "value is CPU",
+                      default="CPU", type=str, metavar='"<device>"')
+    args.add_argument('--loop', default=False, action='store_true',
+                         help='Optional. Enable reading the input in a loop.')
+    args.add_argument("--no_show",
+                      help="Optional. Don't show output",
+                      action='store_true')
+    args.add_argument('-o', '--output', required=False,
+                         help='Optional. Name of output to save.')
+    args.add_argument('-limit', '--output_limit', required=False, default=1000, type=int,
+                         help='Optional. Number of frames to store in output. '
+                              'If 0 is set, all frames are stored.')
+    args.add_argument('-u', '--utilization_monitors', default='', type=str,
+                         help='Optional. List of monitors to show initially.')
+
+    return parser
+
+
+def preprocess_image(image, w, h):
+    image = cv2.resize(image, (w, h))
+    # Change input shape to [B,C,W,H] for MTCNN
+    image = image.transpose((2, 1, 0))
+    image = np.expand_dims(image, axis=0)
+    return image
+
+
+def main():
+    metrics = PerformanceMetrics()
+
+    args = build_argparser().parse_args()
+
+    # Plugin initialization for specified device and load extensions library if specified
+    log.info("Creating Inference Engine")
+
+    ie = IECore()
+
+    # Read IR
+    log.info("Loading network files:\n\t{}".format(args.model_pnet))
+    p_net = ie.read_network(args.model_pnet, args.model_pnet.with_suffix('.bin'))
```

Comment:
Why did you update `read_network()` call with second arg?

Reply:
It was made to support previous version of OV, but I realized, that it bad practice.